policyengine_us/parameters/gov/states/co/ccap/re_determination_hhs_smi_rate.yaml

```diff
@@ -0,0 +1,139 @@
+description: Colorado limits its Child Care Assistance Program re-determination to filers with agi lower than these rates of household size state median income by county.
+metadata:
+  unit: /1
+  period: year
+  label: Colorado Child Care Assistance Program household size state median income re-determination rate
+  reference:
+    - title: COLORADO CHILD CARE ASSISTANCE PROGRAM RULES AND REGULATIONS - 3.103 DEFINITIONS - “Exit income eligibility level”
+      href: https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=6
+    - title: CountyFPGs-CCCAP
+      href: https://docs.google.com/spreadsheets/d/1WzobLnLoxGbN_JfTuw3jUCZV5N7IA_0uvwEkIoMt3Wk/edit#gid=1350122430
+ADAMS_COUNTY_CO:
```

Comment:
@MaxGhenis the value is the same for each county but is reported separately, should we keep it in this breakdown to make county adjustments possible?

Reply:
The rules don't separate by county so we shouldn't either